Bio-page.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alexander Beaumier - Personal Website</title>
+    <link rel="stylesheet" href="css/Styles2.css"> 
+</head>
+<body>
+
+    <!-- Navigation -->
+    <nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="Experience-page.html">Experience</a></li>
```

Comment:
while I see the benefit of having the nav bar change to different nav items depending on what page you're on, I feel like an end user might not pick up on this immediately - (I didn't, and almost missed your amazing fun facts!) Consider a navbar with consistent items so that any page can be easily reached at any point throughout the site.